daprdocs/content/en/getting-started/quickstarts/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+type: docs
+title: "Dapr Quickstarts"
+linkTitle: "Dapr Quickstarts"
+weight: 70
+description: "Try out Dapr quickstarts with code samples that are aimed to get you started quickly with Dapr"
+no_list: true
+---
+
+Hit the ground running with our Dapr quickstarts, complete with code samples aimed to get you started quickly with Dapr.
+
+{{% alert title="Note" color="primary" %}}
+ We are actively working on adding to our quickstart library. In the meantime, you can explore Dapr through our [tutorials]({{< ref "getting-started/tutorials/_index.md" >}}).
+
+{{% /alert %}}
+
+
+#### Before you begin
+
+- [Set up your local Dapr environment]({{< ref "install-dapr-cli.md" >}}).
+
+## Quickstarts
+
+| Quickstarts | Description |
+| ----------- | ----------- |
+| [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Get started with Dapr's Publish and Subscribe building block. |
+| State Management   | Coming soon. |
```

Comment:
we *could* consider pointing to the readme's in the quickstarts repo as we wait for your far better doc to get created.  thoughts?

Reply:
Sounds good to me!

Reply:
Yes, that' a great idea. At least there are example for them to try out while they wait.